### PR TITLE
Remove old travis comments

### PR DESCRIPTION
**Pull request by orta**
You only care about https://github.com/DefinitelyTyped/dt-mergebot/commit/7d11db22bc245cf4216c81840426bb306b9f6969

Commenting inline

### CODE REVIEW

File: src/execute-pr-actions.ts [hunk 1]
```diff
@@ -153,6 +157,26 @@ async function getMutationsForComments(actions: Actions, pr: PR_repository_pullR
...
+    if (!parsed) continue
+
+    if (parsed.tag.includes("travis") && parsed.tag !== travisMessageToKeep?.tag) {
+      mutations.push( createMutation(deleteComment, { input: { id: comment.id } }) )
+    }
+  }
+
+  return mutations;
+}
+
```
Comment (orta) [inline src/execute-pr-actions.ts:179]: Basically find all bot comments about travis which aren't the same as a potential message we might be posting

**Comment by andrewbranch** [comment]
What’s the motivation for this exactly? Is this to clean up from something that went wrong, or is it useful long-term? Can’t quite identify the scenario.

**Comment by orta** [comment]
I don't think we should be leaving a trail of useless comments about old travis results behind in a PR mainly

**Comment by orta** [comment]
I'm gonna merge, if we think it's annoying then a less `--force` approach to is to minimize comments instead. With danger it got annoying real fast when making a new comment each time.

**Pull request merged**